### PR TITLE
Sprints/nineteendo

### PR DESCRIPTION
**Pull request by pierluc-ssoucy**

### CODE REVIEW

File: src/Compliance.Solution/Data/Lookups/data.xml [hunk 1]
```diff
@@ -1,5 +1,20 @@
 
 <entities xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" timestamp="2021-09-08T19:50:52.8881242Z">
+  <entity name="team" displayname="Team">
```
Comment (pierluc-ssoucy) [inline src/Compliance.Solution/Data/Lookups/data.xml:3]: @mitchkirady did we mention that we wanted business units and the other teams in the seed data?

File: src/Compliance.Solution/Solution/Entities/opc_RiskAssessmentDefinition/Entity.xml [hunk 2]
```diff
@@ -609,7 +609,7 @@
             <displayname description="Évaluation des risques" languagecode="1036" />
           </displaynames>
           <Descriptions>
-            <Description description="Reference to the Risk Assessment." languagecode="1033" />
+            <Description description="Référence à l’évaluation des risques." languagecode="1033" />
```
Comment (pierluc-ssoucy) [inline src/Compliance.Solution/Solution/Entities/opc_RiskAssessmentDefinition/Entity.xml:612]: this one strikes back after attempting to do the change from the ui. @pierluc-ssoucy

File: src/Compliance.Solution/Solution/Other/Relationships.xml [hunk 3]
```diff
@@ -338,6 +338,10 @@
   <EntityRelationship Name="lk_opc_topic_createdby" />
   <EntityRelationship Name="lk_opc_topic_modifiedby" />
   <EntityRelationship Name="Mailbox_Annotation" unmodified="1" />
+  <EntityRelationship Name="msdyn_aifptrainingdocument_Annotations" />
+  <EntityRelationship Name="msdyn_aimodel_Annotations" />
+  <EntityRelationship Name="msdyn_aiodimage_Annotations" />
+  <EntityRelationship Name="msdyn_knowledgearticletemplate_SharePointDocumentLocations" />
```
Comment (pierluc-ssoucy) [inline src/Compliance.Solution/Solution/Other/Relationships.xml:344]: @mitchkirady they keep sneaking back in

File: src/Compliance.Solution/Solution/Entities/opc_RiskAssessmentDefinition/Entity.xml [hunk 4]
```diff
@@ -609,7 +609,7 @@
...
             <Description description="Renvoi à l’évaluation des risques." languagecode="1036" />
```
Comment (JonathanThomassin) [inline src/Compliance.Solution/Solution/Entities/opc_RiskAssessmentDefinition/Entity.xml:613]: The fix should have been on the french description.

File: src/Compliance.Solution/Solution/Other/Relationships/opc_RiskAssessment.xml [hunk 5]
```diff
@@ -17,7 +17,7 @@
     <ReferencingAttributeName>opc_riskassessmentid</ReferencingAttributeName>
     <RelationshipDescription>
       <Descriptions>
-        <Description description="Reference to the Risk Assessment." languagecode="1033" />
+        <Description description="Référence à l’évaluation des risques." languagecode="1033" />
         <Description description="Renvoi à l’évaluation des risques." languagecode="1036" />
```
Comment (JonathanThomassin) [inline src/Compliance.Solution/Solution/Other/Relationships/opc_RiskAssessment.xml:21]: Wrong fix.

File: src/Compliance.Solution/Solution/Connectors/opc_5Fpowerautomate-20admin_policytemplateinstances.json [hunk 6]
```diff
@@ -0,0 +1 @@
+null
```
Comment (JonathanThomassin) [inline src/Compliance.Solution/Solution/Connectors/opc_5Fpowerautomate-20admin_policytemplateinstances.json:1]: ?

File: src/Compliance.Solution/Solution/Other/Relationships/opc_RiskAssessment.xml [hunk 7]
```diff
@@ -17,7 +17,7 @@
     <ReferencingAttributeName>opc_riskassessmentid</ReferencingAttributeName>
     <RelationshipDescription>
       <Descriptions>
-        <Description description="Reference to the Risk Assessment." languagecode="1033" />
+        <Description description="Référence à l’évaluation des risques." languagecode="1033" />
```
Comment (pierluc-ssoucy) [inline src/Compliance.Solution/Solution/Other/Relationships/opc_RiskAssessment.xml:20]: @pierluc-ssoucy change me

File: src/Compliance.Solution/Solution/Other/Solution.xml [hunk 8]
```diff
@@ -480,39 +482,50 @@
...
-      <RootComponent type="92" id="{d7c4e2bb-b659-ec11-8f8e-000d3a0a0df6}" behavior="0" />
       <RootComponent type="92" id="{dd130f6e-e6f6-eb11-94ef-000d3a09e9d9}" behavior="0" />
+      <RootComponent type="92" id="{de9ad115-5688-ec11-8d20-0022486dcd27}" behavior="0" />
       <RootComponent type="92" id="{e6299ae6-99b9-eb11-8236-000d3a84e4d3}" behavior="0" />
       <RootComponent type="92" id="{e960b879-1816-ea11-a811-000d3af46489}" behavior="0" />
+      <RootComponent type="92" id="{eb3bc59e-7d80-ec11-8d20-0022486dcca1}" behavior="0" />
       <RootComponent type="92" id="{eb922592-a05e-ec11-8f8e-000d3a0a099d}" behavior="0" />
       <RootComponent type="92" id="{f3203194-dfd9-eb11-bacb-000d3af464df}" behavior="0" />
       <RootComponent type="92" id="{fd563f0d-1010-ec11-b6e6-000d3af4f9a4}" behavior="0" />
+      <RootComponent type="372" id="{c00e6562-48ae-4d01-9009-6174ff1fe55e}" schemaName="opc_5Fpowerautomate-20admin" behavior="0" />
```
Comment (pierluc-ssoucy) [inline src/Compliance.Solution/Solution/Other/Solution.xml:528]: try to rename, and temporarily install connector on base config until Microsoft resolves this.

File: src/Compliance.Package/Deployment/ComplianceDeployment.cs [hunk 9]
```diff
@@ -87,11 +87,15 @@ class ComplianceDeployment : SolutionDeployment
             new OwnerTeam { TeamName = "Compliance - TL Investigations Review Team", TeamId = new Guid("c31cf3e3-c4e2-eb11-bacb-0022486d87ef"), RoleIds = new Guid[]{ new Guid("c15bdc0f-69b2-eb11-8236-000d3af3d03c") }, AADTeams = new string[]{ "Compliance - TL Investigator" } },
         };
 
+        /*
+        // Removed - _restrictedOwnerTeam property no longer used
```
Comment (EvoPheonix) [inline src/Compliance.Package/Deployment/ComplianceDeployment.cs:91]: This commented out code should be deleted instead

File: src/Compliance.Solution/Solution/Connectors/opc_5Fpowerautomate-20admin_policytemplateinstances.json (hunk 6 above)
Comment (pierluc-ssoucy) [reply src/Compliance.Solution/Solution/Connectors/opc_5Fpowerautomate-20admin_policytemplateinstances.json:1]: no idea, part of the connector's package. Microsoft said its a preview feature.

File: src/Compliance.Solution/Solution/Other/Solution.xml (hunk 8 above)
Comment (pierluc-ssoucy) [reply src/Compliance.Solution/Solution/Other/Solution.xml:528]: did not rename, but added to base config.

File: src/Compliance.Package/Deployment/ComplianceDeployment.cs (hunk 9 above)
Comment (pierluc-ssoucy) [reply src/Compliance.Package/Deployment/ComplianceDeployment.cs:91]: agreed, we are still testing, we'll remove once finalized.

File: src/Compliance.Solution/Data/Lookups/data.xml (hunk 1 above)
Comment (mitchkirady) [reply src/Compliance.Solution/Data/Lookups/data.xml:3]: for now still testing the import of one team before adding them all.

**Pull request merged**